src/loader/README.md

```diff
@@ -16,6 +16,44 @@ version 0.8.0 will have timescaledb-0.8.0.so loaded.
    schedulers for any database that has TimescaleDB installed.
    This is done by a background task called the launcher.
 
+# Messages the launcher may receive
+The launcher implements a simple message queue to be notified when
+it should take certain actions, like starting or restarting a
+scheduler for a given database. Note that on startup of the server
+or on creation of a database, the launcher will attempt to start 
+a scheduler for each database, as it cannot figure out whether a 
+scheduler should exist for a given database. The scheduler is 
+responsible for shutting down if it should not exist.
+
```

Comment:
`the launcher will attempt to start 
a scheduler for each database. The scheduler is 
responsible for shutting down if it should not exist, i.e.-- if the database does not have the TimescaleDB extension installed.`